docs/docs/your-fable-project/author-a-fable-library.md

```diff
@@ -18,7 +20,89 @@ The last point may sound complicated but it's only a matter of adding a couple o
 </ItemGroup>
 ```
 
-That's all it takes to make your library compatible with Fable! In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+In order to publish the package to Nuget check [the Microsoft documentation](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli) or alternatively you can also [use Fake](https://fake.build/dotnet-nuget.html#Creating-NuGet-packages).
+
+## Make your package discoverable
+
+[Fable.Packages](https://fable.io/packages/) is a tool making it easy for users to search for Fable packages. 
+
+To make your packages listed on Fable.Packages, you need to add some tags to your `.fsproj`.
+
+<ul class="textual-steps">
+
+<li>
+
+**All Fable** packages must have the `fable` tag.
+
+</li>
+
+<li>
+
+Specify what kind of package you are publishing.
+
+A fable package can be one of the following:
+
+- `fable-library`: A library that can be used in Fable
+
+    Examples of libraries could be [Fable.Promise](https://github.com/fable-compiler/fable-promise/), [Elmish](https://elmish.github.io/), [Thoth.Json](https://thoth-org.github.io//Thoth.Json/), [Feliz](https://zaid-ajaj.github.io/Feliz/)
+
+    <div></div> <!-- Force a space to improve visual -->
+
+- `fable-binding`: The package consist of a set of API to make a native library available
```

Comment:
Is this supposed to be for "pure" bindings? If that's the case we can write: "The package only contains interfaces to make a native library available. These packages don't need to include F# sources."

Reply:
This is indeed supposed to be for "pure" bindings.

I think it is ok if the package adds a few helpers if needed.

For example, I consider the following package to be bindings:

- `Fable.Browser.X`
- `Fable.ReactDom.Types`
- Most of Feliz packages like `Feliz.Recharts`, `Feliz.Markdown` are bindings

So Feliz packages doesn't contains only interfaces, they use classes to map the NPM API. Do we consider your phrasing better even if it doesn't cover Feliz packages here?